web/redirect.html

```diff
@@ -16,43 +14,181 @@
+          <label for="remember">Always do this</label>
+          <div class="checkbox-toggle"></div>
+        </div>
+      </div>
+      <p>If you do not have the LBRY app, you can <a class="button--primary" href="https://lbry.io/get" rel="noopener noreferrer" target="_blank">download it here</a></p>
     </section>
   </body>
 
   <script>
     (function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","https://www.google-analytics.com/analytics.js","ga");
     ga("create", "UA-60403362-9", "auto");
     ga("send", "pageview");
-    
-    const site = window.location.protocol + "//" + window.location.host;
-    const url = decodeURIComponent(window.location.href.replace(site,"").replace(/^\//,""));
-  
-    if (url) {
-      setTimeout(() => {
-        window.location = "lbry://" + url;
-      }, 2000);
-    }
+
+    var site = window.location.protocol + "//" + window.location.host;
+    var url = decodeURIComponent(window.location.href.replace(site,"").replace(/^\//,""));
+
+    (function  letsGo() {
+      const tvPrefix = 'https://beta.lbry.tv/';
+      const appPrefix = 'lbry://';
+
+      let go = true;
+      let storedDest = localStorage.getItem("lbrytarget"); //webButton or appButton
+
+      let choice = storedDest || "webButton";
+      let prefix = appPrefix;
+
+      // elements
+      const dest = document.getElementById('dest');
+      const timer = document.getElementById('timer');
+      const appButton = document.getElementById("appButton");
+      const webButton = document.getElementById("webButton");
+      const rememberBox = document.getElementById("remember");
+      const stopButton = document.getElementById("stopButton");
+      const stopDiv = document.getElementById("stopDiv");
+      const countdown = document.getElementById("countdown");
+      const chooseweapon = document.getElementById("chooseweapon");
+
+      // event listeners
+      appButton.addEventListener("click", handleChoice);
+      webButton.addEventListener("click", handleChoice);
+      stopButton.addEventListener("click", stop);
+      rememberBox.addEventListener("click", handleRemember);
+
+      dest.innerHTML = "LBRY.tv";
+
+      if (storedDest) {
+        rememberBox.checked = true;
+        if (storedDest === "webButton") {
+          prefix = tvPrefix;
+        } else {
+          prefix = appPrefix;
+        }
+      }
+
+      function handleRemember() {
+        stop();
+        !rememberBox.checked && localStorage.removeItem('lbrytarget');
+        !rememberBox.checked && console.log('clearing preference');
+      }
+
+      function handleChoice(event) {
+        stop();
+        choice = event.target.id;
+        dest.innerHTML = (choice === "appButton") ? "your LBRY App" : "LBRY.tv";
```

Comment:
"appButton" and "webButton" should be consts if we're [abiding the maxims](https://gist.github.com/lyoshenka/0a43205aa9a072b196ff87e2c689a8b9)